runtime/src/tests.rs

```diff
@@ -148,19 +142,119 @@ pub fn one_percent() -> Fixed128 {
 	Fixed128::recip(&Fixed128::from_natural(100)).unwrap()
 }
 
-pub fn create_pool() -> DispatchResult {
-	MarginLiquidityPools::create_pool(origin_of(&POOL::get()))
+pub fn negative_one_percent() -> Fixed128 {
+	Fixed128::recip(&Fixed128::from_natural(-100)).unwrap()
 }
 
 pub fn multi_currency_balance(who: &AccountId, currency_id: CurrencyId) -> Balance {
 	<Runtime as synthetic_protocol::Trait>::MultiCurrency::free_balance(currency_id, &who)
 }
 
+pub fn synthetic_create_pool() -> DispatchResult {
+	SyntheticLiquidityPools::create_pool(origin_of(&POOL::get()))?;
```

Comment:
why create twice?

Reply:
Do test cases that have multiple pools. I'll remove it first.